invenio_rdm_records/resources/serializers/bibtex/schema.py

```diff
@@ -46,7 +46,6 @@ class BibTexSchema(BaseSerializerSchema, CommonFieldsMixin):
         # Publication fields
         "publication-conferencepaper": [
```

Comment:
While we're here, how about adding 
```suggestion
        "publication-conferenceproceeding": [BibTexFormatter.proceedings],
```